moduleframework/helpers/openshift_helper.py

```diff
@@ -0,0 +1,268 @@
+            for ns in ['svc', 'dc', 'is']:
+                self._remove_apps_from_openshift_namespaces(ns)
+
+    def _create_app(self):
+        """
+        It creates an application in OpenShift environment
+        """
+        # Switching to system user
+        oc_new_app = self.runHost("oc new-app %s --name=%s" % (self.container_name,
+                                                               self.app_name),
+                                  ignore_status=True)
+        common.print_info(oc_new_app.stdout)
+        time.sleep(1)
+
+    def _verify_pod(self):
+        """
+        It verifies if an application POD is initiated and ready for testing
+        :return: False, application is not initiated during 10 seconds
+                 True, application is initiated and ready for testing
+        """
+        pod_initiated = False
+        for x in range(0, 20):
+            pod_state = self.runHost("oc get pods", ignore_status=True)
+            pod_state = pod_state.stdout.split('\n')
+            for pod in pod_state:
+                if pod.startswith(self.app_name):
+                    if "Running" in pod and "deploy" not in pod:
+                        pod_initiated = True
+                        break
+            time.sleep(1)
+            if pod_initiated:
+                break
+        return pod_initiated
+
+    def setUp(self):
+        """
+        It is called by child class and it is same methof as Avocado/Unittest has. It prepares environment
+        for docker testing
+        * start docker if not
+        * pull docker image
+        * setup environment from config
+        * run and store identification
+
+        :return: None
+        """
+        self.icontainer = self.get_url()
+
+    def _openshift_login(self, oc_ip="127.0.0.1", oc_user='developer', oc_passwd='developer', env=False):
+        """
+        It logins to an OpenShift environment on specific IP and under user and his password.
+        :param oc_ip: an IP where is an OpenShift environment running
+        :param oc_user: an username under which we can login to OpenShift environment
+        :param oc_passwd: a password for specific username
+        :param env:
+        :return:
+        """
+        if env:
+            if 'OPENSHIFT_IP' in os.environ:
+                oc_ip = os.environ.get('OPENSHIFT_IP')
+            if 'OPENSHIFT_USER' in os.environ:
+                oc_user = os.environ.get('OPENSHIFT_USER')
+            if 'OPENSHIFT_PASSWORD' in os.environ:
+                oc_passwd = os.environ.get('OPENSHIFT_PASSWORD')
+
+        oc_output = self.runHost("oc login %s:8443 --username=%s --password=%s" % (oc_ip,
+                                                                                   oc_user,
+                                                                                   oc_passwd),
+                                 verbose=common.is_not_silent())
+        common.print_debug(oc_output.stdout)
```

Comment:
why you use this debugging? isn't better to use `verbose=variable` directly. it exactly does what you want
 `verbose=is_debug()` print via avocado run command output to logger and it is there. in case DEBUG is not set, it is not printed out. (when you don't use it with avocado runner, log is not shown, but it can be enabled.)